feat: update proto to 1.7.0 (#5643)
open-telemetry/opentelemetry-js: e2e-tests/README.md

# OpenTelemetry JS End-to-End Tests

This directory contains end-to-end (E2E) tests for the OpenTelemetry JavaScript project. These tests verify the integration of OpenTelemetry components with a real OpenTelemetry Collector.

## Prerequisites

- [Node.js](https://nodejs.org/) (version as required by the root project)
- [Docker](https://www.docker.com/) (for running the OpenTelemetry Collector)

## Installation

Install dependencies:

```sh
npm install
```

## Running Tests

> The default Make target runs the collector in docker, executes the tests, and verifies the results:

Reviewer: ```suggestion
The default `test:e2e` script runs the collector in docker, executes the tests, and verifies the results:
```

```sh

> make

Reviewer: ```suggestion
npm run test:e2e
```

```

> ### Additional Make Targets

Reviewer: ```suggestion
### Additional npm scripts
```
